### DISCUSSION TITLE
Merge conflict

### DISCUSSION THREAD

**Discussion by ShayBox** [actions/checkout#937] [Q&A] [answered]
https://github.com/ShayBox/ShayBox/tree/master/.github/workflows  
https://github.com/ShayBox/ShayBox/actions/runs/3170734489/jobs/5163533693  

How do I fix these merge conflict errors with actions/checkout@v3?  
There shouldn't be anything to conflict, it's just updating a single file...

**Comment by christianfrstorm** [comment]
--
All the best
Christian Fr stormyhr  Monday, 03 October 2022, 02:47am +02:00 from Shayne Hartford  ***@***.*** :

**Comment by ShayBox** [comment, marked as answer]
I figured it out, you have to force push
